src/components/Pages/Trending/Trending.js

```diff
@@ -0,0 +1,47 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import CustomPagination from "../../Pagination/CustomPagination";
+import SingleContent from "../../SingleContent/SingleContent";
+import "./Trending.css";
+
+const Trending = () => {
+  const [page, setPage] = useState(1);
+  const [content, setContent] = useState([]);
+
+  const fetchTrending = async () => {
+    const { data } = await axios.get(
+      `https://api.themoviedb.org/3/trending/all/day?api_key=1e37d8deae00c9b519356f5d9412edbb&page=${page}`
+    );
+    // console.log(data);
+    setContent(data.results);
+  };
+
+  useEffect(() => {
+    window.scroll(0, 0);
+    fetchTrending();
+    // eslint-disable-next-line
```

Comment:
Why are you disabling eslint here?

Reply:
Without disabling eslint. Prettier says use Effect has a missing dependency.